runtime/near-vm-runner/src/errors.rs

```diff
@@ -65,10 +65,31 @@ impl IntoVMError for wasmer_runtime::error::ResolveError {
 
 impl IntoVMError for wasmer_runtime::error::RuntimeError {
     fn into_vm_error(self) -> VMError {
+        use near_vm_errors::WasmTrap::{BreakpointTrap, GenericTrap};
+        use wasmer_runtime::error::InvokeError;
         use wasmer_runtime::error::RuntimeError;
         match &self {
-            RuntimeError::Trap { msg: _ } => VMError::FunctionCallError(WasmUnknownError),
-            RuntimeError::Error { data } => {
+            RuntimeError::InvokeError(invoke_error) => match invoke_error {
```

Comment:
Which of these lines is handling the case of a trap occurring inside the host function?

Reply:
For the record. Discussing it ATM with Wasmer and @olonho in Slack.